graveler/pgiterator_test.go

```diff
@@ -36,7 +37,7 @@ func TestPGRepositoryIterator(t *testing.T) {
 		}
 		iter.Close()
 
-		if !reflect.DeepEqual(repoIds, []graveler.RepositoryID{"a", "aa", "b", "c", "d", "e"}) {
+		if diffs := deep.Equal(repoIds, []graveler.RepositoryID{"a", "aa", "b", "c", "d", "e"}); diffs != nil {
 			t.Fatalf("got wrong list of repo IDs")
```

Comment:
```suggestion
			t.Fatalf("got wrong list of repo IDs", diffs)
```
`deep.Equal` doesn't help if you don't log it :shrug: 

Also below.

Reply:
added, thanks!